src/modules/rtpengine/rtpengine.c

```diff
@@ -2176,6 +2184,11 @@ static int mod_init(void)
 		}
 	}
 
+	/* Enable ping timer if interval is positive */
+	if(rtpengine_ping_interval > 0) {
+		register_timer(rtpengine_ping_check_timer, 0, rtpengine_ping_interval);
```

Comment:
question: after this timer callback is registered, when will it trigger first time? Is it right away, or after rtpengine_ping_interval seconds?

Reply:
I can't find documentation about it but it seems to be to be after interval seconds and not right away.